modules/representation_management/spec/serializers/accredited_entities/individual_serializer_spec.rb

```diff
@@ -0,0 +1,127 @@
+  subject { described_class.new(individual) }
+
+  before do
+    create(:accredited_individual,
+           :with_organizations,
+           individual_type: 'representative',
+           first_name: 'Bob',
+           last_name: 'Law',
+           address_line1: '123 East Main St',
+           address_line2: 'Suite 1',
+           address_line3: 'Address Line 3',
+           address_type: 'DOMESTIC',
+           city: 'My City',
+           country_name: 'United States of America',
+           country_code_iso3: 'USA',
+           province: 'A Province',
+           international_postal_code: '12345',
+           state_code: 'ZZ',
+           zip_code: '12345',
+           zip_suffix: '6789',
+           phone: '222-222-2222',
+           email: 'email@example.com')
+  end
+
+  let(:individual) do
+    AccreditedIndividual
+      .includes(:accredited_organizations)
+      .first
+  end
+  let(:data) { subject.serializable_hash.with_indifferent_access['data'] }
+  let(:attributes) { data['attributes'] }
+
+  it 'includes id' do
+    expect(data['id']).to eq(individual.id)
+  end
+
+  it 'includes individual_type' do
+    expect(attributes['individual_type']).to eq('representative')
+  end
+
+  it 'includes first_name' do
+    expect(attributes['first_name']).to eq('Bob')
+  end
+
+  it 'includes last_name' do
+    expect(attributes['last_name']).to eq('Law')
+  end
+
+  it 'includes address_line1' do
+    expect(attributes['address_line1']).to eq('123 East Main St')
+  end
+
+  it 'includes address_line2' do
+    expect(attributes['address_line2']).to eq('Suite 1')
+  end
+
+  it 'includes address_line3' do
+    expect(attributes['address_line3']).to eq('Address Line 3')
+  end
+
+  it 'includes address_type' do
+    expect(attributes['address_type']).to eq('DOMESTIC')
+  end
+
+  it 'includes city' do
+    expect(attributes['city']).to eq('My City')
+  end
+
+  it 'includes country_name' do
+    expect(attributes['country_name']).to eq('United States of America')
+  end
+
+  it 'includes country_code_iso3' do
+    expect(attributes['country_code_iso3']).to eq('USA')
+  end
+
+  it 'includes province' do
+    expect(attributes['province']).to eq('A Province')
+  end
+
+  it 'includes international_postal_code' do
+    expect(attributes['international_postal_code']).to eq('12345')
+  end
+
+  it 'includes state_code' do
+    expect(attributes['state_code']).to eq('ZZ')
+  end
+
+  it 'includes zip_code' do
+    expect(attributes['zip_code']).to eq('12345')
+  end
+
+  it 'includes zip_suffix' do
+    expect(attributes['zip_suffix']).to eq('6789')
+  end
+
+  it 'includes phone' do
+    expect(attributes['phone']).to eq('222-222-2222')
+  end
+
+  it 'includes email' do
+    expect(attributes['email']).to eq('email@example.com')
+  end
+
+  it 'includes accredited_organizations' do
+    expect(attributes['accredited_organizations']).not_to be_nil
+  end
+
+  it 'includes all three individual types' do
```

Comment:
I'm curious if there's anything in the serializer to need to test the different types?

Reply:
@holdenhinkle wanted to make sure that the individual types were being passed through for presentation on the front end and this was an easy way to confirm that.

Reply:
It makes more sense to me to do that type of test on the query model to show the results can contain mixed types. The serializer doesn't care about the type.

Reply:
I added a test on the query model to check if all `individual_types` are returned but I'm not sure I understand what you mean here.  I don't see how the query model involves the types at all, it doesn't refer to it in the code and the types aren't involved in querying the database.  In this test I'm making sure that serializer is accurately communicating data that is present on the model.  How does that involve the query?

Reply:
This is my bad. I misunderstood what Holden wanted. You can ignore my suggestion and the test on line 38 should suffice for making sure the type is serialized.